src/main/resources/org/jenkinsci/plugins/junitrealtimetestreporter/RealtimeJUnitStep/help-parseInterval.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <p>
+        The interval between 2 file system scans to parse test results (in milliseconds).
+        Must be a positive value greater than 0.
+        Default value is calculated as 1/100 of the estimated build duration, but never more often than every 5 seconds.
```

Comment:
milliseconds seems to be an odd unit for this field? would you ever actually want to do it that low?